ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivDistributeRawTransaction.java

```diff
@@ -60,35 +60,38 @@ public String getName() {
 
   @Override
   public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
-    final PrivateTransaction privateTransaction;
-    try {
-      privateTransaction = privacySendTransaction.validateAndDecodeRequest(requestContext);
-    } catch (ErrorResponseException e) {
-      return e.getResponse();
-    }
+    final String rawPrivateTransaction = requestContext.getRequiredParameter(0, String.class);
 
-    final SendTransactionResponse sendTransactionResponse;
     try {
-      sendTransactionResponse =
-          privacyController.sendTransaction(
-              privateTransaction, enclavePublicKeyProvider.getEnclaveKey(requestContext.getUser()));
+      final PrivateTransaction privateTransaction =
+          PrivateTransaction.readFrom(RLP.input(Bytes.fromHexString(rawPrivateTransaction)));
+
+      final String enclavePublicKey =
+          enclavePublicKeyProvider.getEnclaveKey(requestContext.getUser());
+      final ValidationResult<TransactionInvalidReason> validationResult =
+          privacyController.validatePrivateTransaction(privateTransaction, enclavePublicKey);
+      if (!validationResult.isValid()) {
+        return new JsonRpcErrorResponse(
+            requestContext.getRequest().getId(),
+            convertTransactionInvalidReason(validationResult.getInvalidReason()));
+      }
+
+      final String enclaveKey =
+          privacyController.sendTransaction(privateTransaction, enclavePublicKey);
+      return new JsonRpcSuccessResponse(
+          requestContext.getRequest().getId(), hexEncodeEnclaveKey(enclaveKey));
     } catch (final MultiTenancyValidationException e) {
       LOG.error("Unauthorized privacy multi-tenancy rpc request. {}", e.getMessage());
       return new JsonRpcErrorResponse(requestContext.getRequest().getId(), ENCLAVE_ERROR);
+    } catch (final IllegalArgumentException | RLPException e) {
+      return new JsonRpcErrorResponse(requestContext.getRequest().getId(), DECODE_ERROR);
     } catch (final Exception e) {
       return new JsonRpcErrorResponse(
-          requestContext.getRequest().getId(),
-          JsonRpcEnclaveErrorConverter.convertEnclaveInvalidReason(e.getMessage()));
+          requestContext.getRequest().getId(), convertEnclaveInvalidReason(e.getMessage()));
     }
+  }
 
-    return privacySendTransaction.validateAndExecute(
-        requestContext,
-        privateTransaction,
-        sendTransactionResponse.getPrivacyGroupId(),
-        () ->
-            new JsonRpcSuccessResponse(
-                requestContext.getRequest().getId(),
-                Bytes.wrap(Base64.getDecoder().decode(sendTransactionResponse.getEnclaveKey()))
-                    .toHexString()));
+  private String hexEncodeEnclaveKey(final String enclaveKey) {
```

Comment:
It's base64 string in the PMT. Why this returns hex don't know, this is what is what already doing. Presumably because values are normally hex encoded in Ethereum RPCs.